packages/lake/src/components/LakeSelect.tsx

```diff
@@ -402,4 +416,8 @@ export function LakeSelect<V>({
       </Popover>
     </View>
   );
-}
+};
+
+export const LakeSelect = forwardRef(LakeSelectWithRef) as <I>(
+  props: SelectProps<I> & { forwardedRef?: ForwardedRef<View> },
```

Comment:
Don't rename the prop, only the variable inside the component

Reply:
🤦